Marlin/example_configurations/delta/kossel_xl/Configuration.h

```diff
@@ -350,15 +312,16 @@
 //===========================================================================
 
 /**
- * Thermal Protection protects your printer from damage and fire if a
+ * Thermal Runaway Protection protects your printer from damage and fire if a
  * thermistor falls out or temperature sensors fail in any way.
  *
  * The issue: If a thermistor falls out or a temperature sensor fails,
  * Marlin can no longer sense the actual temperature. Since a disconnected
  * thermistor reads as a low temperature, the firmware will keep the heater on.
  *
- * If you get "Thermal Runaway" or "Heating failed" errors the
- * details can be tuned in Configuration_adv.h
+ * The solution: Once the temperature reaches the target, start observing.
+ * If the temperature stays too far below the target (hysteresis) for too long,
+ * the firmware will halt as a safety precaution.
  */
```

Comment:
The changes in this comments about Thermal Protection aren't by purpose. Aren't they.